opt-tools/docker/images/app/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11-jre-slim
+CMD cd ../../../../; mvn package;
```

Comment:
This is a not good practice to use `CMD` for change directory. You should use `WORKDIR` for such purposes.

Reply:
Okay, I didn't know, thank you.